vnext/Microsoft.ReactNative.ComponentTests/CommonReaderTest.h

```diff
@@ -0,0 +1,324 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "winrt/Microsoft.ReactNative.h"
+
+namespace winrt::Microsoft::ReactNative::ReaderTestCases {
+
+// a construction to assert JSValueType::Object without hard-coded property order
+
+inline void TestCheckObject(std::map<hstring, std::function<void()>> props, IJSValueReader &reader) {
+  std::set<hstring> propNames;
+  hstring propertyName;
+
+  for (int i = 0; i < props.size(); i++) {
```

Comment:
>int [](start = 7, length = 3)

size_t. I guess it gives a warning on x64.

Reply:
I will fix it, but it doesn't give a warning, I don't know why.